src/containers/shared/components/Transaction/AMMDelete/Description.tsx

```diff
@@ -0,0 +1,34 @@
+import { Trans } from 'react-i18next'
+import { type AMMDelete } from 'xrpl'
+import { TransactionDescriptionProps } from '../types'
+import Currency from '../../Currency'
+
+export const Description = ({
+  data,
+}: TransactionDescriptionProps<AMMDelete>) => {
+  const { Asset, Asset2 } = data.tx
+
+  return (
+    <div data-test="amm-delete-description">
+      <Trans
+        i18nKey="amm_delete_description"
+        components={{
+          Asset: (
+            <Currency
+              currency={Asset.currency}
+              issuer={(Asset as any).issuer}
```

Comment:
If you want to have this error when this is fixed in xrpl.js, you can just use `ts-expect-error` on this line